src/crypto/BUILD.gn

```diff
@@ -25,6 +25,8 @@ if (chip_crypto == "") {
       current_os == "zephyr" || current_os == "mbed" || current_os == "webos" ||
       current_os == "cmsis-rtos") {
     chip_crypto = "mbedtls"
+  } else if (current_os == "linux") {
+    chip_crypto = "mbedtls"
```

Comment:
Can we try to set `chip_crypto` during building the release artifacts, and left `openssl` here?

Reply:
OK, let's check that :)

Reply:
https://github.com/ArekBalysNordic/sdk-connectedhomeip/actions/runs/4193817009/jobs/7271181306